packages/suite/src/components/suite/StakingFeature.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { H3, P, variables } from '@trezor/components';
+
+const Circle = styled.div`
+    border: 9px solid ${({ theme }) => theme.BG_LIGHT_GREEN};
```

Comment:
```suggestion
    border: 9px solid ${({ theme }) => theme.BG_SECONDARY};
```
Design: for now to make it visible

Reply:
Done. But `BG_SECONDARY` is still the same color as `BG_LIGHT_GREEN` with the light mode.

https://github.com/everstake/trezor-suite/pull/1/commits/0ab37330a86b5d543f927db9918649711973f60b

Reply:
Aha, so if you want, just use some different color so that the outside circle is there. But it is just a nitpick and we will have to change it after redesign anyway